apps/rest_api/lib/controllers/admin/user.ex

```diff
@@ -0,0 +1,29 @@
+defmodule RestAPI.Controller.Admin.User do
+  @moduledoc false
+
+  use RestAPI.Controller, :controller
+
+  alias RestAPI.Ports.{Authenticator, ResourceManager}
+  alias RestAPI.Views.Admin.User
+
+  action_fallback RestAPI.Controllers.Fallback
+
+  def create(conn, %{"password" => password} = params) do
+    password_hash = Authenticator.generate_hash(password, :argon2)
```

Comment:
I think we should check password before hashing and everythin where [using this guy](https://github.com/lcpojr/watcher_ex/blob/83c40c401e10ec3c2f3d5be7413e451dc23230dc/apps/resource_manager/lib/credentials/passwords.ex#L15).

In order to avoid calling it directly we can use the port and mock the request.